src/chrome/content/rules/GitLab.io.xml

```diff
@@ -2,20 +2,20 @@
 	For other GitLab coverage, see GitLab.com.xml.
 -->
 
-	<ruleset name="GitLab Pages">
+<ruleset name="GitLab Pages">
+
 	<target host="gitlab.io"/>
 	<target host="*.gitlab.io"/>
 
-	<test url="http://www.gitlab.io/"/>
-	<test url="http://pages.gitlab.io/"/>
-	<test url="http://maxlaumeister.gitlab.io/"/>
+		<test url="http://www.gitlab.io/"/>
+		<test url="http://pages.gitlab.io/"/>
+		<test url="http://maxlaumeister.gitlab.io/"/>
+		<test url="http://moshmage.gitlab.io/" />
+		<test url="http://weel.gitlab.io/" />
 
 	<securecookie host=".+" name=".+"/>
 
-	<!-- (6/2016) GitLab.io doesn't support TLS on 4th level domains.
-		See http://docs.gitlab.com/ee/pages/README.html#limitations -->
```

Comment:
Right. I forgot *again* that left-hand wildcards can be more than one level deep. I will try and look if that comment is still valid.